Formula/pdnsrec.rb

```diff
@@ -12,40 +12,49 @@ class Pdnsrec < Formula
     sha256 "a9faf9edf0e71de5e9ec9fa70d27811f353a8451b41746208fbdc0d592aa5910" => :mountain_lion
   end
 
+  head do
+    url "https://github.com/powerdns/pdns.git"
+
+    depends_on "automake" => :build
+    depends_on "autoconf" => :build
+    depends_on "libtool"  => :build
+    depends_on "pandoc"   => :build
+    depends_on "ragel"    => :build
+  end
+
   depends_on :macos => :lion
   depends_on "boost"
   depends_on "lua" => :optional
 
+  depends_on "pkg-config" => :build if build.with? "lua"
+
   def install
-    # Set overrides using environment variables
-    ENV["DESTDIR"] = "#{prefix}"
-    ENV["OPTFLAGS"] = "-O0"
-    ENV.O0
+    args = %W[
+      --prefix=#{prefix}
+      --sysconfdir=#{prefix}/etc/powerdns
```

Comment:
pdnsrec's `make install` does not write recursor.conf to etcdir, only recursor.conf-dist. Below, at line 47, I actually removed the clobbering that this formula did. If I use #{etc} here, users will have to edit their config in /usr/local/Cellar/... instead of /usr/local/etc/powerdns, as far as I can see.